packages/builders/__tests__/messages/embed.test.ts

```diff
@@ -14,26 +12,6 @@ const base = {
 };
 
 describe('Embed', () => {
-	describe('Embed getters', () => {
-		test('GIVEN an embed with specific amount of characters THEN returns amount of characters', () => {
-			const embed = new EmbedBuilder({
-				title: alpha,
-				description: alpha,
-				fields: [{ name: alpha, value: alpha }],
-				author: { name: alpha },
-				footer: { text: alpha },
-			});
-
-			expect(embedLength(embed.toJSON())).toEqual(alpha.length * 6);
-		});
-
-		test('GIVEN an embed with zero characters THEN returns amount of characters', () => {
-			const embed = new EmbedBuilder();
-
-			expect(embedLength(embed.toJSON(false))).toEqual(0);
-		});
-	});
-
```

Comment:
I'd say we should keep this test, import embedLength from /util. Since this doesn't (only) test the embedLength function but also that it works correctly with EmbedBuilder